python/pyspark/sql/tests/connect/test_parity_functions.py

```diff
@@ -90,8 +90,6 @@ def test_nested_higher_order_function(self):
     def test_np_scalar_input(self):
         super().test_np_scalar_input()
 
-    # TODO(SPARK-41904): Fix nth_value value
-    @unittest.skip("Fails in Spark Connect, should enable.")
     def test_nth_value(self):
```

Comment:
is this PR related to https://github.com/apache/spark/pull/39456?

Reply:
Yes. The bug will duplicate the fields for output.